docker/ubuntu-2004-builder.Dockerfile

```diff
@@ -72,6 +72,8 @@ RUN <<EOT bash
     rm \$CLANGPATH.tar.xz
     mv \$CLANGPATH clang+llvm-15.0.6
 EOT
+ENV LLVM_CONFIG_PATH=/opt/clang+llvm-15.0.6/bin/llvm-config
```

Comment:
Is this needed if it's in PATH?

Reply:
I'm not sure, but this PR just replicates the existing behavior (the older builder image currently used by cicd also has it in the path, and still passes it as an explicit var to the docker image).

Should I try removing it?

Reply:
It's not clear to me why it was added in the first place.
image-builders:
```
commit 5e9cf7b832db69880f3c5bd0fb9a5263da51695a
Author: Todd Fleming <tbfleming@gmail.com>
Date:   Mon May 9 19:05:25 2022 -0400

    add clang-14
```
psibase:
```
commit 0e7e6d38eed30e7ae33cf703782074387d22fd49
Author: Todd Fleming <tbfleming@gmail.com>
Date:   Wed May 11 18:45:12 2022 -0400

    Use libclang instead of clang-doc
```

According to https://github.com/KyleMayes/clang-sys, it should be okay to find in PATH.